### ISSUE TITLE
Scale Docker Container

### ISSUE THREAD

**Issue by vany0114** [Microsoft/service-fabric#130]
Hi guys,

I wanted to ask what is the best way to scale Docker containers in Service Fabric, I mean, I want to gain density per node, having multiple instances of the same container per node, but I only was able to scale one instance per node. I tried to partition the service but it didn't work, so I went with `SingletonParition`, but I really would like to have multiple instances of the same container per node.

Thanks in advance.

**Comment by vany0114** [comment]
I don't know, I haven't able to figure it out. I have a cluster with 5 nodes, and this is the configuration for one of the service, in this case, I'm trying to deploy at least 2 instances (PartitionCount="2" ) per node.

```
<Service Name="Invoice" ServicePackageActivationMode="ExclusiveProcess">
      <StatelessService ServiceTypeName="InvoiceType" InstanceCount="[Invoice_InstanceCount]">
        <UniformInt64Partition PartitionCount="2" LowKey="1" HighKey="5" />
      </StatelessService>
</Service>
```
I have serveral erros: 
![2018-05-21_19-48-29](https://user-images.githubusercontent.com/12032625/40336562-ef05afc6-5d2f-11e8-8173-fa4ca38f2056.png)
![2018-05-21_19-43-27](https://user-images.githubusercontent.com/12032625/40336507-9a5a5f8a-5d2f-11e8-8a6f-c0db69b8f777.png)
![2018-05-21_19-30-17](https://user-images.githubusercontent.com/12032625/40336508-9a6f505c-5d2f-11e8-8bcb-76ab417bfa60.png)

I really would appreciate your help

**Comment by masnider** [comment]
Depending on how your docker container is actually set up regarding networking and communication, the most common issue that causes this is a port conflict, where both container instances are trying to claim the same host port. @vany0114 did you ever get this sorted out?

**Comment by vany0114** [comment]
Yeah, effectively it was a port conflict, I had a container which runs a web api thru a static port, let's say 5080, so I can't create several containers in the same node running over the same port, so I think I would need a reverse proxy to do that. So, you guys have some example showing how to use the SF reverse proxy to achieve that?
